content/blogposts/2022/ranking-in-a-week.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'Ranking in a week'
+description: 'TODO'
+authors:
+  - name: Eric Fritz
+    url: https://eric-fritz.com
+publishDate: 2022-11-17T12:00+00:00
+tags: [blog]
+slug: ranking-in-a-week
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/TODO.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/TODO.png
```

Comment:
@iskyOS Will need a new hero image here.

Reply:
Will have this Friday

Reply:
```suggestion
heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/ranking/page-rank-tech-bog-illustration-final.png
socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/ranking/page-rank-tech-bog-illustration-final.png
```